src/modules/main-nav/MainNav.js

```diff
@@ -4,6 +4,33 @@ import { mainNav } from './../../data/home.json'
 
 export default class MainNav {
   constructor (node) {
+    this.node = node
     this.node.innerHTML = template(mainNav)
+    this.eventButtonMenu()
+    this.eventOptionMenu()
+  }
+
+  eventButtonMenu () {
+    const buttonMenu = this.node.querySelector('.button-closed')
+    const mainNav = this.node.querySelector('.main-nav-menu')
+    const buttonCross = this.node.querySelector('.button-closed-line')
+    console.log(mainNav)
+    buttonMenu.addEventListener('click', function () {
+      mainNav.classList.toggle('main-nav-menu--opened')
+      buttonCross.classList.toggle('button-closed-line')
+      buttonCross.classList.toggle('button-closed-cross')
```

Comment:
`button-closed-cross` and `button-closed-line` don't look like modifiers and they should probably be.